tests/test_dense.py

```diff
@@ -55,3 +55,15 @@ def test_trace(self, matrix, trace):
         cupy_array = CuPyDense(matrix)
 
         assert cupy_array.trace() == trace
+
+
+def test_no_checks_constr():
+    """Checks if CuPyDense canbe properly inherited"""
```

Comment:
```suggestion
    """Checks if CuPyDense can be properly inherited"""
```

Reply:
I'm not really sure what this is testing? I.e. in what circumstances do we expect to wrap ``CuPyDense`` and how might it fail?

E.g. I would never expect the code below to fail because it's only using basic Python semantics -- this is sub-class of a pure Python class testing that the inherited methods exist.

There are things that I would expect to fail -- e.g. ``wrapcpd`` hasn't been registered with the data layer, but I think we should just delete this test for now and revisit it when we have something more concrete to test.

Reply:
This test was purely to debug the custom constructor in case we needed some kind of inheritance, I am erasing it.